pkg/envtest/crd.go

```diff
@@ -115,8 +115,8 @@ func InstallCRDs(config *rest.Config, options CRDInstallOptions) ([]*apiextensio
 	return options.CRDs, nil
 }
 
-// readCRDFiles reads the directories of CRDs in options.Paths and adds the CRD structs to options.CRDs.
-func readCRDFiles(options *CRDInstallOptions) error {
+// ReadCRDFiles reads the directories of CRDs in options.Paths and adds the CRD structs to options.CRDs.
+func ReadCRDFiles(options *CRDInstallOptions) error {
```

Comment:
Not sure if I should change the input paramter to only take / return exactly what is needed.

The current input parameter is a bit awkward. On the other side we have the same in WaitForCRDs

Reply:
No strong opinion either way

Reply:
Yeah same :)